plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/access/DBAAuthModel.java

```diff
@@ -97,4 +97,12 @@ void refreshCredentials(
         @NotNull CREDENTIALS credentials
     ) throws DBException;
 
+    /**
+     * Getting necessity of the hostname field
+     *
+     * @return necessity of the hostname field
+     */
+    default boolean isHostRequired() {
+        return true;
```

Comment:
Host is not required by many JDBC drivers. In Generic driver we can verify this by presence of {host} item in sample URL